src/core/math/quat.js

```diff
@@ -536,6 +554,43 @@ class Quat {
         return this;
     }
 
+    /**
+     * Set the quaternion that represents the shortest rotation from one direction to another.
+     *
+     * @param {Vec3} from - The direction to rotate from. It should be normalized.
+     * @param {Vec3} to - The direction to rotate to. It should be normalized.
+     * @returns {Quat} Self for chaining.
+     *
+     * Proof of correctness: https://www.xarg.org/proof/quaternion-from-two-vectors/
```

Comment:
```suggestion
     * {@link https://www.xarg.org/proof/quaternion-from-two-vectors/ Proof of correctness}
```

Reply:
I haven't checked this works....but just thought it'd be cool to actually hyperlink this.